mlrun/runtimes/serving.py

```diff
@@ -241,6 +241,7 @@ def set_tracking(
         :param stream_args:  stream initialization parameters, e.g. shards, retention_in_hours, ..
         """
         self.spec.track_models = True
+
```

Comment:
Seems redundant

Reply:
How is this redundant? This is client side code for setting the model monitoring on...

Reply:
I meant for the newline addition 😂